package.json

```diff
@@ -28,6 +28,7 @@
     "express": "^4.17.1",
     "joi": "^14.3.1",
     "jsonwebtoken": "^8.5.1",
+    "lodash": "^4.17.15",
     "mocha": "^6.2.0",
     "nodemon": "^1.19.2",
     "nyc": "^14.1.1",
```

Comment:
@tuyisengepaul , lock your dependencies and dev-dependencies by removing all caret(^) symbols before the dependency version. this ensure that the same version of the module is installed whichever year you return to this project. hence it attracts no breaking changes due to file upgrade

Reply:
Resolved